api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -34,11 +34,7 @@
 					<property name="target">
 						<bean class="${project.parent.groupId}.${project.parent.artifactId}.api.impl.ObsServiceImpl">
 							<property name="dao">
-								<bean class="${project.parent.groupId}.${project.parent.artifactId}.api.db.hibernate.HibernateFHIRDAO">
```

Comment:
In order to limit the changes to the absolute minimum necessary to add support for retrieving orders by accession number, i would use a separate pull request for the changes in this file.